ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-traffic-engineering.j2

```diff
@@ -0,0 +1,65 @@
+{# eos - Router Traffic Engineering #}
+{% if router_traffic_engineering is arista.avd.defined %}
+!
+router traffic-engineering
+{%     if router_traffic_engineering.segment_routing is arista.avd.defined %}
+{%         set ns = namespace(delimiter=false) %}
+   segment-routing
+{%         if router_traffic_engineering.segment_routing.colored_tunnel_rib is arista.avd.defined(true) %}
+{%             set ns.delimiter = true %}
+      rib system-colored-tunnel-rib
+{%         endif %}
+{%         if router_traffic_engineering.segment_routing.policy_endpoints is arista.avd.defined %}
+{%             for endpoint in router_traffic_engineering.segment_routing.policy_endpoints | arista.avd.natural_sort %}
+{%                 for color in endpoint.colors | arista.avd.natural_sort %}
+{%                     if ns.delimiter is arista.avd.defined(true) %}
+      !
+{%                     endif %}
+{%                     set ns.delimiter = true %}
+      policy endpoint {{ endpoint.address }} color {{ color.value }}
+{%                     if color.binding_sid is arista.avd.defined %}
+         binding-sid {{ color.binding_sid }}
+{%                     endif %}
+{%                     if color.name is arista.avd.defined %}
+         name {{ color.name }}
+{%                     endif %}
+{%                     if color.description is arista.avd.defined %}
+         description {{ color.description }}
+{%                     endif %}
+{%                     for pathgroup in color.path_group | arista.avd.natural_sort %}
+         !
+         path-group preference {{ pathgroup.preference }}
+{%                         if pathgroup.explicit_null is arista.avd.defined("ipv4") %}
+{%                             set expnull = "ipv4" %}
+{%                         endif %}
+{%                         if pathgroup.explicit_null is arista.avd.defined("ipv6") %}
+{%                             set expnull = "ipv6" %}
+{%                         endif %}
+{%                         if pathgroup.explicit_null is arista.avd.defined("ipv4 ipv6") %}
+{%                             set expnull = "ipv4 ipv6" %}
+{%                         endif %}
+{%                         if expnull is arista.avd.defined %}
```

Comment:
We don't need to check the value in ```pathgroup.explicit_null```. Also this doesn't consider the value **none** though that is a valid user input. 

So please remove lines 32-40.

Modify ->
line 41: ```if pathgroup.explicit_null is arista.avd.defined```
line 42: ```explicit-null {{ pathgroup.explicit_null }}```